droopler.info.yml

```diff
@@ -109,6 +105,6 @@ themes:
 
 distribution:
   name: 'Droopler Profile'
-version: '3.1.0'
+version: '3.2.0-rc1'
```

Comment:
should we change version now or later only in release branch?

Reply:
It is perfectly OK to change it here.
If you get a package from drupal.org, this field is filled automatically.
However, on Github we must update it manually